app/views/pages/covid_19.html.erb

```diff
@@ -14,16 +14,5 @@
 
     <p>For AACT database users, a table called "covid_19_studies" containing the 68 columns from the spreadsheet, is now included within AACT.</p>
```

Comment:
Hey @ievgend2 there are some users that scrape this page for the links, could you still include this html? You can hide it using css like `display: none`